build-system/tasks/presubmit-checks.js

```diff
@@ -48,6 +48,7 @@ var forbiddenTerms = {
     message: requiresReviewPrivacy,
     whitelist: [
       'src/cid.js',
+      'src/service/cid-impl.js',
```

Comment:
awesome, i was just thinking yesterday wanting to suggest we add a service folder :+1: